### DISCUSSION TITLE
Добавить возможность убрать названия хедеров типа "кнопка" и пространство, которое они занимают

### DISCUSSION THREAD

**Discussion by simplepersonru** [totumonline/totum-ru-issues-and-discussions#412] [Как это сделать?] [answered]
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/71624171/13c06736-73ed-4833-955a-d7a56f709d38)

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/71624171/a7b4e5a1-3197-4c88-a339-b0b0391ccd62)

Как правило всегда в хедере кнопке хочется не видеть название хедера, потому что хочется его видеть в кнопке, как это сделано в любой графической библиотеке \ фреймворке в той или иной компутерной технологии.

Как видно из картинок, вне зависимости от того есть название у хедера или нет, место он занимает и прилично - примерно столько же сколько сама кнопка.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Смотрите Секции параметр title: false или title: fildname: false.

он же в setFloatFormat форматирования поля (но это не точно)

**Reply by simplepersonru** [reply]
Секция и title: false - то что доктор прописал!
